crypto/ec/asm/ecp_sm2p256-armv8.pl

```diff
@@ -28,44 +28,44 @@
 sub bn_mod_add() {
 	my $mod = shift;
 $code.=<<___;
-	# Load inputs
+	// Load inputs
```

Comment:
Should these be C style comments, rather than c++ style?  Or does the MSVC compiler not recognize the former?

Reply:
I simply kept the end-of-line style (C++/C99). It is already used in that file, e.g.
https://github.com/openssl/openssl/blob/8a7f30ef67d59f808610275e8bc1723510cf062b/crypto/ec/asm/ecp_sm2p256-armv8.pl#L167

If you care about style, you might be more interested in what becomes `// ### xxxx ###`.

Reply:
This is a reasonable way to do things. The Perl translation programs output assembler in `.S` files, and these ones are always fed to a C99+ compiler - we use `//`-style comments elsewhere, but the `#`s get parsed by the pre-processor (`.S` rather than `.s` file) so yes, can cause problems